dcs/src/main/resources/dcs-default.xml

```diff
@@ -386,4 +386,12 @@
         Timeout minutes between first and max times (6 default) DCS Server startup MXOSRVR.
     </description>
   </property>
+  <property>
+    <name>dcs.server.user.program.keepalive.opt</name>
+    <value>ENABLE=enable,IDLETIME=300,INTERTIME=5,RETRYCNT=3</value>
```

Comment:
Can we split this into multiple property/values and  change the property name 
 dcs.server.user.program.tcp.keepalive.status
dcs.server.user.program.tcp.keepalive.idletime
dcs.server.user.program.tcp.keepalive.intervaltime
dcs.server.user.program.tcp.keepalive.retrycount